persim/pers_landscape.py

```diff
@@ -0,0 +1,72 @@
+"""
+    Base Persistence Landscape class.
+
+    authors: Gabrielle Angeloro, Michael Catanzaro
+"""
+from abc import ABC, abstractmethod
+import numpy as np
+
+
+class PersLandscape(ABC):
```

Comment:
How is having the `PersLandscape` base class beneficial? It looks like in all the implementations, you implement `__init__` anyways.

Reply:
It currently houses repeated code/checks from the two classes (the `__init__` calls `super()` in addition to other stuff), but it may have made more sense to have in an older version of our project. We can remove and refactor if you'd like.